### ISSUE TITLE
Numbers shown do not match up in any meaningful way

### ISSUE THREAD

**Issue by cbondurant** [wordpressenginetracker/wordpressenginetracker.github.io#30]
Hi, I was looking at this page in order to better understand the state of WP Engine.

I noticed something strange with the numbers that are listed in site-data.json.

It contains two values for every day beginning on September 21st, one which is the current total number of sites using WP engine, and a second which is the number of sites that have moved *off* WP Engine.

Curiously the *total* number of sites decreases at a substantially greater rate than *moved* increases. Just under twice as quickly in fact. There is no explanation on the site for this discrepancy.

Is it that the additional loss in sites is a result of sites moving off of wordpress entirely due to the hostile actions taken by the CEO of wordpress? Having some clarification would be helpful for all involved. I am sure that no one wishes to misinform or mislead.

Additionally I notice that the chart has some strange resizing on it, that results in the bottom of the chart not being zero. I would recommend that this be fixed as it really does lead to many people getting the wrong impression and thinking that almost a third of WPEngine's users have left. Not that I could tell you how much the correct percentage is, on account of all these other issues with the data that I have pointed out.

**Comment by studiochampgauche** [comment]
This site is fake and is made by losers. They need 3 heads for render a single Hero/splash section, because instead of having learn their job, they have suck Matt's dick. 🫤

**Comment by studiochampgauche** [comment]
This site is made by losers. They need 3 heads for render a single Hero/splash section, because instead of having learn their job, they have suck Matt's dick. 🫤

**Comment by studiochampgauche** [comment]
If you think about it for a moment, you'll realize that they needed 4 heads to create one section. So you know the level is low.

**Comment by wordpressenginetracker** [comment]
Hi @cbondurant 

> the total number of sites decreases at a substantially greater rate than moved increases.

Yep this is correct. "Moved" sites are sites (and the domain) which has been confirmed to no longer be on WPE but is still a live site that is hosted elsewhere. There are other reasons why the total number of sites can decrease. The most common one is sites that have completely gone offline (about 13k total right now). We've also removed some number of staging sites, domains that got redirected (though sometimes those get added back in depending on how the redirect works), and there are typically a few thousand sites that have various errors at any time (sometimes related to them being in the process of moving or going offline).

> Is it that the additional loss in sites is a result of sites moving off of wordpress 

Nope. The sites that moved may or may not still be on WordPress. But we've looked at this and only about 2% of the moved sites seem to have moved off of WordPress. Discussed a bit here: https://github.com/wordpressenginetracker/wordpressenginetracker.github.io/issues/27#issuecomment-2492525327
